### ISSUE TITLE
Dump generator no longer seems to work properly on FANDOM sites

### ISSUE THREAD

**Issue by huckleberrypie** [WikiTeam/wikiteam#330]
Seems like this got whacked up ever since Wikia made the wise decision of transitioning into FANDOM. Executing python dumpgenerator.py --index=https://americangirl.wikia.com/w/index.php --xml --images results in a 404; changing wikia to fandom goes a bit further but I get a timeout.

**Comment by nemobis** [comment]
Huck Jones, 03/12/18 12:45:
> Executing python dumpgenerator.py 
> --index=https://americangirl.wikia.com/w/index.php --xml --images 

Why? You should use
	--api=http://americangirl.wikia.com/api.php --xml --xmlrevisions

**Comment by huckleberrypie** [comment]
I got a bit further but I end up with this error:
File "dumpgenerator.py", line 2323, in <module>
    main()
  File "dumpgenerator.py", line 2313, in main
    resumePreviousDump(config=config, other=other)
  File "dumpgenerator.py", line 1966, in resumePreviousDump
    config=config, titles=titles, session=other['session'])
  File "dumpgenerator.py", line 745, in generateXMLDump
    for xml in getXMLRevisions(config=config, session=session):
  File "dumpgenerator.py", line 800, in getXMLRevisions
    site = wikitools.wiki.Wiki(config['api'])
NameError: global name 'wikitools' is not defined

**Comment by nemobis** [comment]
Huck Jones, 03/12/18 13:06:
> I got a bit further but I end up with this error:

Yes, as the help says you need to install the wikitools package.

**Comment by nemobis** [comment]
The one thing that wasn't working was the image format. Fixed in 17283113dda2d5553ef5d65fe83f9afe13e55307

Example:
```
$ ls acrossingfandomcom_ca-20200210-wikidump/images/
01013 flow 2560x1600.JPG                              Animal Forest.jpg.desc       Button redirection copiar.png                           Chico.png.desc
01013 flow 2560x1600.JPG.desc                         Asteriscos.gif               Button redirection copiar.png.desc                      Chico-render(verano).png
Abstractos 65-t2.JPG                                  Asteriscos.gif.desc          Caca.jpg                                                Chico-render(verano).png.desc
Abstractos 65-t2.JPG.desc                             Avatar.png                   Caca.jpg.desc                                           Contenido.png
ACE(Invierno)(2).PNG                                  Avatar.png.desc              Captura de pantalla 2013-05-25 a les 15.41.03.png       Contenido.png.desc
ACE(Invierno)(2).PNG.desc                             Banano.gif                   Captura de pantalla 2013-05-25 a les 15.41.03.png.desc  Destacado.png
Añadida por.JPG                                       Banano.gif.desc              Captura de pantalla 2013-05-25 a les 16.50.43.png       Destacado.png.desc
Añadida por.JPG.desc                                  Bandera española.jpg         Captura de pantalla 2013-05-25 a les 16.50.43.png.desc  Edificios.jpg
Animal Crossing Enciclopedia.jpg                      Bandera española.jpg.desc    Casa.jpg                                                Edificios.jpg.desc
Animal Crossing Enciclopedia.jpg.desc                 Barradherramientas.jpg       Casa.jpg.desc                                           Ejemplo.jpg
Animal Crossing Enciclopedia.png                      Barradherramientas.jpg.desc  Categorías Totodilefan.png                              ESBOZO.PNG
Animal Crossing Enciclopedia.png.desc                 Bean monalisa.jpg            Categorías Totodilefan.png.desc                         ESBOZO.PNG.desc
Animal Crossing New Leaf Carátula Americana.jpg       Bean monalisa.jpg.desc       Chica-render(verano).png                                !.jpg
Animal Crossing New Leaf Carátula Americana.jpg.desc  Bubbles.gif                  Chica-render(verano).png.desc                           !.jpg.desc
Animal Forest.jpg                                     Bubbles.gif.desc             Chico.png
```
